libs/platform/src/lib/components/list/list.component.ts

```diff
@@ -404,6 +404,16 @@ export class ListComponent extends CollectionBaseInput implements OnInit, AfterV
             item._noSeperator = this.noSeperator;
             this.stateChanges.next(item);
         });
+
+
+        const indicators = this.itemEl.nativeElement.querySelectorAll('fd-busy-indicator');
+        if (indicators) { 
+            indicators.forEach((indicator) => {
+                if (indicator) {
```

Comment:
`indicators` can be empty or can contain some nodes, so it can not be aka `[null, null]` since so no need to check it.
Please correct me if I'm wrong.